REPOSITORIES.md

```diff
@@ -1,6 +1,7 @@
 - guardian/amiable:dependency-updates_1
 - guardian/amigo:dependency-updates
 - guardian/invoicing-api
+- guardian/janus-app:dependency-updates
```

Comment:
Unfortunately [janus-app](https://github.com/guardian/janus-app) [is now failing](https://github.com/guardian/scala-steward-public-repos/runs/8252271982?check_suite_focus=true#step:5:150) with Scala Steward (tho' weirdly [it _was_ working](https://github.com/guardian/scala-steward-public-repos/runs/8247567489?check_suite_focus=true#step:5:140) for a few hours after this PR was initially merged?!) as is:

```
  [info] [launcher] getting org.scala-sbt sbt 1.7.1  (this may take some time)...
  [info] [launcher] getting Scala 2.12.16 (for sbt)...
  [info] welcome to sbt 1.7.1 (Eclipse Adoptium Java 11.0.16.1)
  [info] loading global plugins from /home/runner/.sbt/1.0/plugins
  [info] compiling 1 Scala source to /home/runner/.sbt/1.0/plugins/target/scala-2.12/sbt-1.0/classes ...
  [info] Non-compiled module 'compiler-bridge_2.12' for Scala 2.12.16. Compiling...
  [info]   Compilation completed in 12.405s.
  [info] done compiling
  [info] loading settings for project janus-app-build from plugins.sbt ...
  [info] loading project definition from /home/runner/scala-steward/workspace/repos/guardian/janus-app/project
  Error:  java.lang.RuntimeException: found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
  Error:  
  Error:  	* org.scala-lang.modules:scala-xml_2.12:2.1.0 (early-semver) is selected over {1.2.0, 1.1.1, 1.0.6}
  Error:  	    +- com.github.sbt:sbt-native-packager:1.9.11 (sbtVersion=1.0, scalaVersion=2.12) (depends on 2.1.0)
  Error:  	    +- org.scala-lang:scala-compiler:2.12.16              (depends on 1.0.6)
  Error:  	    +- com.typesafe.sbt:sbt-native-packager:1.5.2 (scalaVersion=2.12, sbtVersion=1.0) (depends on 1.1.1)
  Error:  	    +- com.typesafe.play:twirl-api_2.12:1.5.1             (depends on 1.2.0)
```

Could this be because `janus-app` has an unusual build setup that involves running the [janus/tc-build.sh](https://github.com/guardian/janus/blob/main/tc-build.sh) build script from an entirely different repo? Maybe Scala Steward has no idea about this, and tries to run sbt without pulling in any appropriate conf (note that it's been run by Scala Steward with sbt 1.7.1, even though [janus-app is actually on 1.6.2](https://github.com/guardian/janus-app/blob/99e4887a4b17f65ea0f22eef926de5aecc482340/project/build.properties) - I assume Scala Stewards sbt is just defaulting to the latest version of sbt).

In any case, maybe best to remove janus-app from the list of repos unless we can figure this out?!

Reply:
Thanks for flagging @rtyley - I checked after merging, but didn't spot the later failures!

I think it's the same cause as we saw for Amigo a while back (https://github.com/guardian/scala-steward-public-repos/pull/9#issuecomment-1206501971). One of the projects plugins' dependencies (after an update was automerged) was deemed to be binary incompatible, so the sbt project itself isn't loadable. It's easy to fix by reverting the offending update and pinning the version, so I'll do that today.

Related thought; it would be great to have a better notification system for failing actions... maybe it's possible to build a small one ourselves that notifies a team rather than whoever last changed the cron.

Reply:
Back to green now 🎉

Reply:
> It's easy to fix by reverting the offending update and pinning the version, so I'll do that today.

Nice one man! I can see the build is passing now - what's the commit that made that fix?

Reply:
revert: https://github.com/guardian/janus-app/commit/2168178fea4b09b4e194a297efa4b51801cc24c8
pinning (and update as far as we can): https://github.com/guardian/janus-app/commit/ca78bfc62f005fd691c571f1aa1f1615f7e674e4